conf/unloader.cfg

```diff
@@ -21,6 +21,8 @@ send_ur = false
 # Only for job records and summaries, to report on local jobs at the site.
 local_jobs = true
 
+# Set to true to withhold DNs from unloaded records. Valid only for job, sync,
+# and cloud records.
```

Comment:
Why would someone want to do this?

Reply:
I've updated the comment.

Reply:
Thanks, much cleared, one new typo now though.